src/dal-plugin/Stream.mm

```diff
@@ -143,10 +143,7 @@ - (CMSimpleQueueRef)copyBufferQueueWithAlteredProc:(CMIODeviceStreamQueueAltered
 }
 
 - (CVPixelBufferRef)createPixelBufferWithTestAnimation {
-    
-    if (NSEqualSizes(self.testCardSize, NSZeroSize)){
-        self.testCardSize = [self testCardImageSize];
-    }
+    self.testCardSize = [self testCardSize];
```

Comment:
this is unnecessary. `self.testCardSize` and `[self testCardSize]` are exactly the same thing! ObjectiveC is weird like that.